ADTTools/DeleteModels/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ADTToolsLibrary;
+using Azure;
+using Azure.DigitalTwins.Core;
+using Azure.Identity;
+using CommandLine;
+using System;
+using System.Linq;
+using System.Net;
+
+namespace DeleteModels
+{
+    class Program
+    {
+        private readonly Options options;
+
+        static void Main(string[] args)
+        {
+            Parser.Default.ParseArguments<Options>(args).WithParsed(options => new Program(options).Run());
+        }
+
+        private Program(Options options)
+        {
+            this.options = options;
+        }
+
+        private void Run()
+        {
+            try
+            {
+                var credential = new InteractiveBrowserCredential(options.TenantId, options.ClientId);
+                var client = new DigitalTwinsClient(new UriBuilder("https", options.HostName).Uri, credential);
+                DeleteAllModels(client, 1);
+            }
+            catch (Exception ex)
+            {
+                Log.Error($"Deleting models failed.");
+                Log.Error(ex.Message);
+            }
+        }
+
+        private void DeleteAllModels(DigitalTwinsClient client, int iteration)

```

Comment:
Does it delete all twins as well? I assume no.

Reply:
No, it only deletes models. The intent is to preserve the same behavior as the -d option in the ModelUploader tool.